pkgdb2/lib/__init__.py

```diff
@@ -370,7 +372,7 @@ def set_acl_package(session, pkg_name, pkg_branch, pkg_user, acl, status,
                 'You are not allowed to approve or deny '
                 'ACLs for yourself.')
 
-    if pkg_user.startswith('group::') and acl == 'approveacls':
+    if '-' in pkg_user and acl == 'approveacls':
```

Comment:
I am not sure whether this is a good replacement for group:: check, since there are a lot of groups without - in their name.